File: INF333/lista-0/A.cpp
#include <iostream>
#include <vector>

int main() {
    int dinners, alumni, presence;

    std::cin >> alumni >> dinners;

    while (dinners != 0 && alumni != 0) {
        std::vector<int> v(alumni, 0);

        for (int i = 0; i < dinners; i++) {
            for (int j = 0; j < alumni; j++) {
                std::cin >> presence;

                v[j] += presence;
            }
        }
        
        bool perfect = false;

        for (int i = 0; i < alumni; i++) {
            if (v[i] == dinners) perfect = true;
        }

        if (perfect) std::cout << "yes\n";
        else std::cout << "no\n";

        std::cin >> alumni >> dinners;
    }

    return 0;
}

File: INF333/lista-0/A.py
while True:
    try:
        alumni, dinners = [int(x) for x in input().strip().split(" ")]

        if dinners == 0 and alumni == 0:
            break

        v = [0] * alumni

        for i in range(0, dinners):
            presence = [int(x) for x in input().strip().split(" ")]
            # print(presence)
            for j in range(0, len(presence)):
                v[j] += presence[j]

        # print(v)

        perfect = False
        for a in v:
            if a == dinners:
                perfect = True

        if perfect:
            print("yes")
        else:
            print("no")

    except EOFError:
        break
